src/components/CalendarMonthGrid.jsx

```diff
@@ -60,6 +64,19 @@ const defaultProps = {
   monthFormat: 'MMMM YYYY', // english locale
 };
 
+function getCalendarMonthGridWidthStyle(orientation, withWeekNumbers) {
```

Comment:
:| I really want to work on the with-styles change more now cos this sucks that we have to do this.